Tipstaff/Infrastructure/RepositoriesInstaller.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Castle.MicroKernel.Registration;
+using Castle.MicroKernel.SubSystems.Configuration;
+using Castle.Windsor;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Tipstaff.Infrastructure.DynamoAPI;
+using Tipstaff.Infrastructure.Repositories;
+using Tipstaff.Infrastructure.S3API;
+using Tipstaff.Infrastructure.Services;
+using Tipstaff.Logger;
+using Tipstaff.Services.Repositories;
+
+namespace Tipstaff.Services.Infrastructure
+{
+    class RepositoriesInstaller : IWindsorInstaller
+    {
+        public void Install(IWindsorContainer container, IConfigurationStore store)
+        {
+            
+            //REPOSITORIES
+            container.Register(Component.For<IApplicantRepository>().ImplementedBy<ApplicantRepository>());
+            container.Register(Component.For<IAttendanceNotesRepository>().ImplementedBy<AttendanceNotesRepository>().LifestylePerWebRequest());
+            container.Register(Component.For<IChildRepository>().ImplementedBy<ChildRepository>());
+            container.Register(Component.For<IContactsRepository>().ImplementedBy<ContactsRepository>());
+            container.Register(Component.For<IDocumentsRepository>().ImplementedBy<DocumentsRepository>());
+            container.Register(Component.For<IFAQRepository>().ImplementedBy<FAQRepository>());
+            container.Register(Component.For<ISolicitorFirmRepository>().ImplementedBy<SolicitorFirmRepository>());
+            container.Register(Component.For<ISolicitorRepository>().ImplementedBy<SolicitorRepository>());
+            container.Register(Component.For<ITemplateRepository>().ImplementedBy<TemplateRepository>());
+            container.Register(Component.For<ITipstaffRecordRepository>().ImplementedBy<TipstaffRecordRepository>());
+            container.Register(Component.For<IDeletedTipstaffRecordRepository>().ImplementedBy<DeletedTipstaffRecordRepository>());
+            container.Register(Component.For<ICaseReviewRepository>().ImplementedBy<CaseReviewRepository>());
+            container.Register(Component.For<IAddressRepository>().ImplementedBy<AddressRepository>());
+            container.Register(Component.For<IRespondentRepository>().ImplementedBy<RespondentRepository>());
+
+
+            //MISC
```

Comment:
I think you've left these as well in the PresentationLayerInstaller.cs file. Is there a need for these to be duplicated?